docs/topics/images.md

```diff
@@ -0,0 +1,364 @@
+(image_tag)=
+# How to use images in templates
+
+The `image` tag inserts an XHTML-compatible `img` element into the page, setting its `src`, `width`, `height` and `alt`. See also [](image_tag_alt).
+
+The syntax for the tag is thus:
+
```

Comment:
a section got lost here

---

    ```html+django
    {% image [image] [resize-rule] %}
    ```

    **Both the image and resize rule must be passed to the template tag.**

    For example: